Inconsistent peerDependency for babel-plugin-lingui-macro

**Describe the bug**
lingui 5.1.0

It seems the versions of babel-plugin-lingui-macro got out of sync.

[@lingui/macro package](https://github.com/lingui/js-lingui/blob/main/packages/macro/package.json)  has a peerDependency:

    "@lingui/babel-plugin-lingui-macro": "4.11.2",

While [@lingui/react package](https://github.com/lingui/js-lingui/blob/main/packages/react/package.json) has a different peerDependency:

    "@lingui/babel-plugin-lingui-macro": "5.0.0",

This is causing NPM to complain:

```
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: @lingui/core@5.1.0
npm warn Found: @lingui/babel-plugin-lingui-macro@5.1.0
npm warn node_modules/@lingui/babel-plugin-lingui-macro
npm warn   @lingui/babel-plugin-lingui-macro@"5.1.0" from @lingui/cli@5.1.0
npm warn   node_modules/@lingui/cli
npm warn     dev @lingui/cli@"^5" from the root project
npm warn     1 more (@lingui/vite-plugin)
npm warn
npm warn Could not resolve dependency:
npm warn peerOptional @lingui/babel-plugin-lingui-macro@"4.11.2" from @lingui/macro@5.1.0
npm warn node_modules/@lingui/macro
npm warn   @lingui/macro@"^5" from the root project
npm warn ERESOLVE overriding peer dependency
```

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -69,7 +69,7 @@
     "unbuild": "2.0.0"
   },
   "peerDependencies": {
-    "@lingui/babel-plugin-lingui-macro": "5.0.0",
+    "@lingui/babel-plugin-lingui-macro": "5.1.0",
     "babel-plugin-macros": "2 || 3"
   },
   "peerDependenciesMeta": {

--- a/packages/macro/package.json
+++ b/packages/macro/package.json
@@ -51,7 +51,7 @@
     "@lingui/react": "5.1.0"
   },
   "peerDependencies": {
-    "@lingui/babel-plugin-lingui-macro": "4.11.2",
+    "@lingui/babel-plugin-lingui-macro": "5.1.0",
     "babel-plugin-macros": "2 || 3"
   },
   "peerDependenciesMeta": {

--- a/packages/react/package.json
+++ b/packages/react/package.json
@@ -77,7 +77,7 @@
     "macro/index.js"
   ],
   "peerDependencies": {
-    "@lingui/babel-plugin-lingui-macro": "5.0.0",
+    "@lingui/babel-plugin-lingui-macro": "5.1.0",
     "babel-plugin-macros": "2 || 3",
     "react": "^16.8.0 || ^17.0.0 || ^18.0.0 || ^19.0.0"
   },

